lib/semian/circuit_breaker.rb

```diff
@@ -89,6 +89,17 @@ def in_use?
       !error_timeout_expired? && !@errors.empty?
     end
 
+    def to_s
```

Comment:
I'd guess you're trying to implement the `inspect` method that is intended to be used during debugging. The `to_s` method is more general purpose and should provide only the information that is meaningful in text context (usually the less, the better to avoid unintentional information leakage if exposed to the user).